drivers/dai/intel/ssp/ssp.c

```diff
@@ -57,28 +57,27 @@ static struct dai_intel_ssp_mn ssp_mn_divider = {
 	.base = DT_REG_ADDR_BY_IDX(DT_NODELABEL(ssp0), 1),
 };
 
-
-#define INTEL_SSP_INST_DEFINE(node_id) {				\
-		.is_initialized = false,				\
-		.is_power_en = false,					\
-		.acquire_count = 0,					\
-		.ssp_index = DT_PROP(node_id, ssp_index),		\
-		.base =	DT_REG_ADDR_BY_IDX(node_id, 0),			\
-		IF_ENABLED(DT_NODE_EXISTS(DT_NODELABEL(sspbase)),	\
-		(.ip_base = DT_REG_ADDR_BY_IDX(DT_NODELABEL(sspbase), 0),))	\
-		.shim_base = DT_REG_ADDR_BY_IDX(DT_NODELABEL(shim), 0),		\
-		IF_ENABLED(DT_NODE_EXISTS(DT_NODELABEL(hdamlssp)),		\
-			(.hdamlssp_base = DT_REG_ADDR(DT_NODELABEL(hdamlssp)),))\
-		IF_ENABLED(DT_PROP_HAS_IDX(node_id, i2svss, 0),			\
-			(.i2svss_base = DT_PROP_BY_IDX(node_id, i2svss, 0),))	\
+#define INTEL_SSP_INST_DEFINE(node_id) {			\
+		.is_initialized = false,			\
+		.is_power_en = false,				\
+		.acquire_count = 0,				\
+		.ssp_index = DT_PROP(node_id, ssp_index),	\
+		.base =	DT_REG_ADDR_BY_IDX(node_id, 0),		\
+		IF_ENABLED(DT_NODE_EXISTS(DT_NODELABEL(sspbase)),			\
+		(.ip_base = DT_REG_ADDR_BY_IDX(DT_NODELABEL(sspbase), 0),))		\
+		.shim_base = DT_REG_ADDR_BY_IDX(DT_NODELABEL(shim), 0),			\
+		IF_ENABLED(DT_NODE_EXISTS(DT_NODELABEL(hdamlssp)),			\
+			(.hdamlssp_base = DT_REG_ADDR(DT_NODELABEL(hdamlssp)),))	\
+		IF_ENABLED(DT_PROP_HAS_IDX(node_id, i2svss, 0),				\
+			(.i2svss_base = DT_PROP_BY_IDX(node_id, i2svss, 0),))		\
```

Comment:
This looks like 10 lines of pure whitespace changes. This is not only very distracting (does it hide any actual change?), it can also break git merge/cherry-pick/revert/etc.

https://wiki.openstack.org/wiki/GitCommitMessages#Things_to_avoid_when_creating_commits